src/Sarif/Core/SarifRewritingVisitor.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Rewriting visitor for the Sarif object model.
```

Comment:
>model [](start = 47, length = 5)

Add a comment explaining that this is file contains the bulk of the changes that need to be "manually" applied to the class, and that it will go away once the class can be fully auto-generated.